CHANGELOG.md

```diff
@@ -4,15 +4,35 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
-## Unreleased
+## 1.7.3 (2020-07-27)
+
+This maintenance release contains bugfixes since the 1.7.2 release. We deem it high
+priority for upgrading.
+
+In particular the fixes contained in this maintenance release address bugs in compression,
```

Comment:
Nit: I suggest saying "...address issues related to compression..."

Saying "bugs" sounds like there was something wrong with compression while in reality we improved handling of things like setting tablespace.